src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/reporting/serviceprovider/performance/PerformanceReportJobListener.kt

```diff
@@ -29,13 +36,25 @@ class PerformanceReportJobListener(
 
   override fun afterJob(jobExecution: JobExecution) {
     if (jobExecution.status == BatchStatus.COMPLETED) {
-      val path = jobExecution.executionContext.get("output.file.path") as String
-
-      s3Service.publishFileToS3(Path(path), "reports/service-provider/performance/")
+      val path = Path(jobExecution.executionContext.getString("output.file.path"))
+
+      s3Service.publishFileToS3(path, "reports/service-provider/performance/")
+
+      emailSender.sendEmail(
+        notifyTemplateId,
+        jobExecution.jobParameters.getString("user.email"),
+        mapOf(
+          "serviceProviderFirstName" to jobExecution.jobParameters.getString("user.firstName"),
+          "reportUrl" to UriComponentsBuilder.fromHttpUrl(interventionsUiBaseUrl)
```

Comment:
I feel that there is sufficient enough complexity here to warrant a unit test.

Reply:
👍